TF_Settings_Web/src/Pages/Visuals/VisualsScreen.tsx

```diff
@@ -92,7 +93,7 @@ const VisualsScreen: React.FC = () => {
     if (!isDesktop() || !hasReadConfig) return <></>;
 
     return (
-        <div className={classes('scroll-div')}>
+        <div className={classes('scroll-div')} onPointerUp={writeConfig}>
```

Comment:
💬 This is going to result in a lot more writes to the config (i.e. spam clicking the background will end up writing)
I don't think this is too big an issue